### DISCUSSION TITLE
Mobile Development

### DISCUSSION THREAD

**Discussion by krunaldodiya** [NativePHP/laravel#184] [Q&A] [answered]
Is there any plan for mobile app development just like React Native ?

**Comment by osc2nuke** [comment]
I remember a long time ago i played around with this:

[Appcelerator on Github](https://github.com/appcelerator-archive/titanium_desktop)
I think that was even before React Native.

Just want to shout-out to all these amazing dev's, especially the ones what made NativePHP!


PS: I only made the reference to above Appcelerator project for inspiration.

**Comment by simonhamp** [comment, marked as answer]
Not for v1
